### DISCUSSION TITLE
Reinstall Home Assistant and Zigbee2MQTT Plug-in w/o pairing?

### DISCUSSION THREAD

**Discussion by hawaltie** [Koenkk/zigbee2mqtt#18645] [General]
I run HA for 3 years, and I want to start over again. In this time frame new features have been released, and I want to make use of it, and start with a clean install. However, I don't like to pair all my devices ( > 30 ) again. Is there a way to avoid this?

**Comment by francisp2** [comment]
Make a copy of your /data directory , or in case you run the HA addon, your /config/zigbee2mqtt directory

**Reply by niek1992** [reply]
Afaik you don't need to kopy the /data directory. As long as the PAN ID is noted down before starting from scratch, all the Zigbee devices should connect again to the coördinator as long as the chipset doesn't change too much.

**Comment by hawaltie** [comment]
Thanx for your response. If possible I prefer the PAN ID only solution, to start from scratch as much as possible.

**Reply by csszokolai** [reply]
Hi! This method is worked for you? Thanks a lot!
